### PR TITLE
Fixed Orch template issues

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 4dcc57f70c6f5ff7a4bd19a204dc2aef0a5313c3:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/services/test_orchestration_template.py [hunk 1]
```diff
@@ -49,12 +49,17 @@ def pytest_generate_tests(metafunc):
...
+    return fauxfactory.gen_alphanumeric()
+
+
 def test_orchestration_template_crud(provisioning):
     template_type = provisioning['stack_provisioning']['template_type']
     template = OrchestrationTemplate(template_type=template_type,
                                      template_name=fauxfactory.gen_alphanumeric(),
                                      description="my template")
-    template.create(METHOD_TORSO)
+    method = METHOD_TORSO.replace('CloudFormation', random_desc())
```
Comment (psav) [inline cfme/tests/services/test_orchestration_template.py]: If we are inserting a randomized string into the description, is there a need to have two ```METHOD_TORSO```s defined now? Also, as ```random_desc()``` does not hold any state etc, I don't see the real need to create a function for running that single line of code. I could be missing something though :)

File: cfme/tests/services/test_orchestration_template.py [hunk 2]
```diff
@@ -77,9 +84,10 @@ def test_name_required_error_validation(provisioning):
     template = OrchestrationTemplate(template_type=template_type,
                                      template_name=None,
                                      description="my template")
+    method = METHOD_TORSO.replace('CloudFormation', random_desc())
 
     with error.expected(flash_msg):
-        template.create(METHOD_TORSO)
+        template.create(method)
```
Comment (psav) [inline cfme/tests/services/test_orchestration_template.py]: Thinking about this, why not create a function scoped fixture that would return a ```METHOD_TORSO``` with a randomized string in the description? That way you have eliminated the duplicate random calls, and making if function scoped will mean you don't need to worry about them colliding.

**Comment by dajoRH** [comment]
Lint report for commit ac7097f604f6088b2f591410b3ad3807c9af8990:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**